integration_tests/client/filtering/test_filtering_examples.py

```diff
@@ -0,0 +1,321 @@
+
+    swimmer = Label(key="class", value="swimmer")
+    boat = Label(key="class", value="boat")
+    fish = Label(key="class", value="fish")
+
+    dataset = Dataset.create("ocean_images")
+    for uid, is_large_boat, is_swimmer in contains_boat_swimmer:
+        dataset.add_groundtruth(
+            GroundTruth(
+                datum=Datum(uid=uid),
+                annotations=[
+                    Annotation(
+                        labels=[boat],
+                        bounding_box=large_box if is_large_boat else small_box,
+                        is_instance=True,
+                    ),
+                    Annotation(
+                        labels=[swimmer if is_swimmer else fish],
+                        bounding_box=small_box,
+                        is_instance=True,
+                    ),
+                ],
+            )
+        )
+
+    # No swimmer, small boats
+    no_swimmer_small_boats = client.get_datums(
+        Filter(
+            datums=And(
+                Label.key == "class",
+                Label.value != "swimmer",
+            ),
+            annotations=And(
+                Label.key == "class",
+                Label.value == "boat",
+                Annotation.bounding_box.area < 50,
+            ),
+        )
+    )
+    assert len(no_swimmer_small_boats) == 1
+    assert no_swimmer_small_boats[0].uid == "uid1"
+
+    # No swimmer, large boats
+    no_swimmer_large_boats = client.get_datums(
+        Filter(
+            datums=And(
+                Label.key == "class",
+                Label.value != "swimmer",
+            ),
+            annotations=And(
+                Label.key == "class",
+                Label.value == "boat",
+                Annotation.bounding_box.area > 50,
+            ),
+        )
+    )
+    assert len(no_swimmer_large_boats) == 1
+    assert no_swimmer_large_boats[0].uid == "uid2"
+
+    # Swimmer with small boat
+    swimmer_with_small_boats = client.get_datums(
+        Filter(
+            datums=And(
+                Label.key == "class",
+                Label.value == "swimmer",
+            ),
+            annotations=And(
+                Label.key == "class",
+                Label.value == "boat",
+                Annotation.bounding_box.area < 50,
+            ),
+        )
+    )
+    assert len(swimmer_with_small_boats) == 1
+    assert swimmer_with_small_boats[0].uid == "uid3"
+
+    # Swimmer with large boat
+    swimmers_and_boats = client.get_datums(
+        Filter(
+            datums=And(
+                Label.key == "class",
+                Label.value == "swimmer",
+            ),
+            annotations=And(
+                Label.key == "class",
+                Label.value == "boat",
+                Annotation.bounding_box.area > 50,
+            ),
+        )
+    )
+    assert len(swimmers_and_boats) == 1
+    assert swimmers_and_boats[0].uid == "uid4"
+
+
+# NOTE The example below does not work.
```

Comment:
commented code?

Reply:
why was this resolved? looks like there is still unnecessary commented code here